docs/MAINTAINERS.adoc

```diff
@@ -0,0 +1,200 @@
+
+=== Ladder
+
+* *Contributors*. Anyone can participate.
+  There is no precondition except having a GitHub account, just submit pull requests or comment in existing ones!
+* **Core Pull Request Reviewers** can be nominated by contributors in the link:https://groups.google.com/d/forum/jenkinsci-dev[developer mailing list]. 
+Self-nomination is fine.
+Decision is made by a consensus in the mailing list or via voting at the link:https://jenkins.io/project/governance-meeting/[governance meeting].
+* **Core Maintainers** - same process as for Core PR reviewers.
+  All nominees must also sign an link:https://github.com/jenkinsci/infra-cla/[Individual Contributor License Agreement] before getting the permission in GitHub repositories.
+* **Release Team Members** are assigned by the Jenkins Release officer
+
+== Pull request review process
+
+Jenkins core is a mission-critical part of the ecosystem.
+We need to ensure that submitted pull requests are not only code complete,
+but also that they do not introduce undesired defects, breaking changes and technical debt.
+At the same time, we are interested to make the review process as simple as possible for contributors and maintainers.
+
+=== Code review goals
+
+There are the following code review goals:
+
+* Ensure that the pull request is feasible and that it fits the Jenkins ecosystem
+* Verify compatibility of the suggested changes
+* Ensure that the pull request matches the code quality practices: test automation, Javadoc for newly introduced APIs, etc.
+* Ensure that the pull request contains appropriate changelogs and, if needed, upgrade guidelines.
+* Ensure there is a consensus among reviewers about all items above 
+
+**Feasibility**.
+Reviewers are expected to look at changes critically from a "product management" point of view.
+It's not just about the code, but also whether the change makes sense in a global / holistic way, considering existing popular plugins and the way users experience Jenkins overall.
+Also, not every change needs to be merged into the core.
+Many features would be better introduced as plugins which have separate release cycles and allow delivering changes faster.
+We want to extend the Jenkins core and incorporate widely used functionality and extension points there,
+but we try to keep the core as minimal as possible in terms of dependencies.
+
+**Compatibility**.
+The Jenkins project has a long history of backward compatibility.
+We accept breaking changes when it is needed (security fixes, feature deprecation and removal, etc.),
+but ultimately the project's goal is retain as much compatibility as possible.
+It includes both feature compatibility and binary/API compatibility which is important for the plugin ecosystem.
+Although we provide some tools for checking API usages across open-source plugins,
+there is no way to confirm external usages in 3rd-party proprietary plugins which is also a part of the ecosystem.
```

Comment:
Would not consider that relevant. It's also not actionable, what would a reader take from this?

Reply:
![image](https://user-images.githubusercontent.com/3000480/74657443-8c58ed00-5190-11ea-9466-bb2f5305ce29.png)